UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/CustomInputLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
+{
+    public abstract class CustomInputLogger : BasicInputLogger
+    {
+        [Tooltip("The data will be saved as CSV files.")]
```

Comment:
Understood you just copied this code from MRTK2...

Can you change this tooltip to a _summary_ block?  As far as I'm aware this field is not shown in the inspector, but maybe I'm wrong